anabeesh/build.gradle

```diff
@@ -42,6 +42,10 @@ android {
 }
 
 dependencies {
+
+    compile 'com.google.code.gson:gson:2.8.2'
+    compile 'com.squareup.retrofit2:converter-gson:2.3.0'
```

Comment:
1- e7na 3andena file lel dependencies esmo `dependencies.gradle` 7ot feeh el 7agat di w e3mel reference hena

2- `compile` di deprecated, estakhdem `implementation`

Reply:
wlahy dwrt we ml2tooosh

Reply:
Project view > tools > gradle

Aw double shift w e3mel search

Reply:
F kol el a7wal lw 3ayez te3mello add keda bera7tK bas knaleeha implementation

Reply:
done . 3mltha zy mnta 3mlha rwsh neek 'D